code/game/objects/items/storage/belt.dm

```diff
@@ -842,6 +842,62 @@
 	new /obj/item/melee/sabre/solgov(src)
 	update_appearance()
 
+/obj/item/storage/belt/sabre/suns
+	name = "SUNS sabre sheath"
+	desc = "A leather sheath designed to hold an blade."
```

Comment:
```suggestion
	desc = "A leather sheath designed to hold a blade."
```
In my opinion, this desc is a little plain - maybe you could do something like...
```suggestion
	desc = "A leather sheath for sabers, commonly worn by SUNS academic staff."
```